_posts/2016-10-31-pulp-3.0-mvp.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Pulp Celerybeat and the Pulp 3.0 Minimum Viable Product (MVP)
+date: 2016-08-18T18:28:49+00:00
+layout: post
+permalink: /2016/08/18/sprint-demo-6-august-18-2016/
+author: Dennis Kliban
+tags:
+  - MVP
+  - 3.0
+  - Celerybeat
+categories:
+  - Uncategorized
+---
+Last week during our [sprint 9 demo]({% post_url 2016-10-27-sprint-demo-9 %}) I gave an update on
+the work I did on Pulp Celerybeat for Pulp 3.0. I'd like to use this post to highlight these
+changes in written form.
+
+Pulp 3.0 is a complete rewrite of Pulp. To make sure that we could accomplish the work in a
+reasonable amount of time we decided to put together a description of a [Minimum Viable Product
+(MVP) for Pulp 3.0.0](https://pulp.plan.io/projects/pulp/wiki/Pulp_3_Minimum_Viable_Product)
+release. This MVP is a subset of the functionality currently found in Pulp. We hope Pulp users will
+take a look at the MVP and provide feedback based on their workflows and how the MVP affects them.
+It is possible that we have overlooked something critical to our users, so please take a look at
```

Comment:
s/to our users, so please take a look at MVP description/to your usage of Pulp, so please send feedback to pulp-list@redhat.com./

^ should be an email link.